thumbnails/docs/getting-started.md

```diff
@@ -0,0 +1,290 @@
+
+THUMBNAILS_FILESYSTEMSTORAGE_ROOT
+: Root path of the filesystem storage directory, defaults to `<os tempdir>/ocis-thumbnails/`
+
+THUMBNAILS_WEBDAVSOURCE_BASEURL
+: Base url for a webdav api, defaults to `https://localhost:9200/remote.php/webdav/`
+
+THUMBNAILS_RESOLUTIONS
+: List of resolutions supported by the service, defaults to `["16x16", "32x32", "64x64", "128x128"]
+
+#### Health
+
+THUMBNAILS_DEBUG_ADDR
+: Address to debug endpoint, defaults to `0.0.0.0:9189`
+
+### Commandline flags
+
+If you prefer to configure the service with commandline flags you can see the available variables below.
+
+#### Global
+
+--config-file
+: Path to config file, empty default value
+
+--log-level
+: Set logging level, defaults to `info`
+
+--log-color
+: Enable colored logging, defaults to `true`
+
+--log-pretty
+: Enable pretty logging, defaults to `true`
+
+#### Server
+
+--tracing-enabled
+: Enable sending traces, defaults to `false`
+
+--tracing-type
+: Tracing backend type, defaults to `jaeger`
+
+--tracing-endpoint
+: Endpoint for the agent, empty default value
+
+--tracing-collector
+: Endpoint for the collector, empty default value
+
+--tracing-service
+: Service name for tracing, defaults to `thumbnails`
+
+--debug-addr
+: Address to bind debug server, defaults to `0.0.0.0:9189`
+
+--debug-token
+: Token to grant metrics access, empty default value
+
+--debug-pprof
+: Enable pprof debugging, defaults to `false`
+
+--debug-zpages
+: Enable zpages debugging, defaults to `false`
+
+--grpc-name
+: Name of the service, defaults to `thumbnails`
+
+--grpc-addr
+: Address to bind grpc server, defaults to `0.0.0.0:9185`
+
+--grpc-namespace
+: Set the base namespace for the grpc namespace", defaults to `com.owncloud.api`
+
+--filesystemstorage-root
+: Root path of the filesystem storage directory, defaults to `<os tempdir>/ocis-thumbnails/`
+
+--webdavsource-baseurl
+: Base url for a webdav api, defaults to `https://localhost:9200/remote.php/webdav/`
+
+--thumbnail-resolution
+: List of resolutions supported by the service, defaults to `["16x16", "32x32", "64x64", "128x128"]
+
+#### Health
+
+--debug-addr
+: Address to debug endpoint, defaults to `0.0.0.0:9189`
+
+### Configuration file
+
+So far we support the file formats `JSON` and `YAML`, if you want to get a full example configuration just take a look at [our repository](https://github.com/owncloud/ocis/thumbnails/tree/master/config), there you can always see the latest configuration format. These example configurations include all available options and the default values. The configuration file will be automatically loaded if it's placed at `/etc/ocis/thumbnails.yml`, `${HOME}/.ocis/thumbnails.yml` or `$(pwd)/config/thumbnails.yml`.
+
+## Usage
+
+The program provides a few sub-commands on execution. The available configuration methods have already been mentioned above. Generally you can always see a formated help output if you execute the binary via `ocis-thumbnails --help`.
+
+### Server
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (116:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)